src/Microsoft.AspNet.StressFramework/StressTestRunner.cs

```diff
@@ -82,14 +82,23 @@ public class StressTestRunner : TestRunner<StressTestCase>
                 StressTestTrace.WriteLine("Launching Host");
 
                 // Launch the host to run the test
-                var host = LaunchHost(TestMethod);
+                var stopwatch = new Stopwatch();
+                using (var host = LaunchHost(TestMethod))
+                {
+                    host.AttachCollectors();
+
+                    StressTestTrace.WriteLine("Releasing Host");
+                    stopwatch.Start();
+                    host.BeginIterations();
+
+                    host.ProcessCollectorsAsync();
```

Comment:
Curious: Why are we not able to attach collectors and then start them immediately?